doc/source/building/build_pxe_root_filesystem.rst

```diff
@@ -6,74 +6,103 @@ Build a PXE root File System Image
 
    build_pxe_root_filesystem/setup_pxe_bootserver
 
-.. hint::
```

Comment:
Better use "Root" instead of "root" in the title "Build a PXE root File System Image"